util/gconv/gconv_z_unit_converter_test.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/gogf/gf/v2/os/gtime"
 	"github.com/gogf/gf/v2/test/gtest"
 	"github.com/gogf/gf/v2/util/gconv"
+	"google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
It will add extra dependent package to `go.mod`. Is it better move this unit testing case to example folder?

Reply:
I think maybe I can create a new struct instead of it.